03-auto_tick.xsh

```diff
@@ -227,7 +227,7 @@ def run(feedstock=None, protocol='ssh',
     body = ('Merge only after success.\n\n'
             'This PR was created by [regro auto-tick](https://github.com/regro/cf-scripts). '
             'Please let the devs know if there are any [issues](https://github.com/regro/cf-scripts/issues). '
-            "Please feel free to push to the bot's branch to update this PR. \n\n")
+            "Please feel to push to the bot's branch to update this PR. \n\n")
```

Comment:
I don't think this is a typo.

Reply:
I'm just very confused, sorry.

Reply:
The point of this message was to encourage maintainers to push to the bot's branch. The bot doesn't usually fix it's own PRs and the bot devs don't have the bandwidth to fix all it's errors or update the recipe.